docs/ApiGuidelines.md

```diff
@@ -46,8 +55,10 @@ Experimental API requirements:
 * Experimental APIs must be in a header file that is published to the SDK output directory. They
   can be in their own header, or in a header file that is shared with public APIs but only
   if surrounded by an appropriate ifdef (e.g., "#ifdef OE\_CONTEXT\_SWITCHLESS\_EXPERIMENTAL\_FEATURE").
-* Experimental APIs should be fully documented using doxygen markup in the header file
-* Experimental APIs must have at least one test case.
+* Experimental APIs should be fully documented using doxygen markup in the header file, and include the @experimental markup.
```

Comment:
Per the discussion in PR [#2579](https://github.com/openenclave/openenclave/pull/2579#discussion_r384115879), do we want to be more prescriptive in the use of the `@experimental` API tag? For example, do we want to recommend that all new APIs should be marked experimental APIs for at least a minor version release?

Reply:
I think this is a good idea. Can we add it to the follow up PR that will address the outstanding comments here?

Reply:
I don't think that serves any purpose pre-1.0. There's no effective compatibility SLA, and staying away from `@experimental` APIs does nothing to change that. If OE decides to break compatibility, that's a minor release, and there's no break-free upgrade path from the previous release at all.